eth/handler.go

```diff
@@ -453,9 +482,16 @@ func (h *handler) BroadcastBlock(block *types.Block, propagate bool) {
 		} else {
 			transfer = peers[:int(math.Sqrt(float64(len(peers))))]
 		}
+		diff := h.chain.GetDiffLayerRLP(block.Hash())
 		for _, peer := range transfer {
+			if len(diff) != 0 && peer.diffExt != nil {
+				// difflayer should send before block
+				log.Error("===debug Broadcast block", "number", block.Number(), "hash", hash)
```

Comment:
remove debug log

Reply:
please ignore this. I add it for testing, and it is not done yet.